src/test/java/com/checkmarx/flow/cucumber/integration/pullrequest/updatecomments/UpdatePullRequestCommentsSteps.java

```diff
@@ -94,20 +111,84 @@ private void initSca() {
         scaProperties.setAccessControlUrl("https://platform.checkmarx.net");
     }
 
+    private class ScanResultsAnswerer implements Answer<ScanResults> {
+
+        private FilterConfiguration filterConfiguration;
+
+        private FilterConfiguration switchFilterConfiguration(){
+            if(filterConfiguration == filterMedium){
+                return filterLow;
+            }
+            else{
+                return filterMedium;
+            }
+        }
+
+        @Override
+        public ScanResults answer(InvocationOnMock invocation) throws CheckmarxException {
+
+            filterConfiguration = switchFilterConfiguration();
+            ScanResults results = cxClientMock.getReportContent(sastFile, filterConfiguration);
+            CxScanSummary summary =  new CxScanSummary();
```

Comment:
inline this like this:
 results.setScanSummary(new CxScanSummary());

Reply:
Fixed